panel/links.py

```diff
@@ -100,12 +102,29 @@ class Callback(param.Parameterized):
     # Whether the link requires a target
     _requires_target = False
 
-    def __init__(self, source, target=None, **params):
+    def __init__(self, source: UnknownType, target: UnknownType=None, args: Dict[str, UnknownType]=None, code: Dict[str, str]=None):
```

Comment:
When you define a custom `__init__` I would argue you have to make the function signature more specific to help the user: `**params` is not very helpful.